test/let_spec.rb

```diff
@@ -0,0 +1,45 @@
+BDDSM.describe 'BDDSM::Describe.let' do # rubocop:disable Metrics/BlockLength
+  # let should define variable accessible from 'it'
+  let(:a) { 1 }
+  it { expect(a).to eq 1 }
+
+  # let variable should be accessible from another let
+  let(:one) { 1 }
+  let(:two) { one + one }
+  it { expect(two).to eq 2 }
+
+  # let variable should be overriden
+  let(:three) { 3 }
+  let(:three) { 4 }
+  it { expect(three).to eq 4 }
+
+  # checks if let block executes only once
+  module Count # rubocop:disable Lint/ConstantDefinitionInBlock
+    module_function
+
+    @instance_count ||= 0
+
+    def instance_count
+      @instance_count
+    end
+
+    def test(counter)
```

Comment:
What this this method is testing? Looks like it increments something

Reply:
ok i'll rename it